docs/resources/endpoint_security_group.md

```diff
@@ -145,52 +217,100 @@ All examples for the Endpoint Security Group resource can be found in the [examp
 * `admin_state` (shutdown) - (string) Withdraw the ESG configuration from all nodes in the fabric.
   - Default: `no`
   - Valid Values: `no`, `yes`.
-
+  
```

Comment:
same as datasources

Reply:
This only appears in the code and not in the actual docs. There are removable though